server/services/GithubWorkflowService.js

```diff
@@ -147,6 +147,12 @@ const createGithubWorkflow = async ({ job, directory, gitSha }) => {
   if (atKey === 'nvda') {
     inputs.test_pattern = '{reference/**,test-*-nvda.*}';
     inputs.browser = browser;
+    inputs.nvda_version = atVersion?.name;
+  }
+  if (atKey === 'voiceover_macos') {
+    // We just want the whole number of the macOS version
+    // due to limitations on Github workflow runners
```

Comment:
Thanks for leaving this comment! Outside the context of this PR, future readers may be curious on which limitation. Could you also link https://github.com/w3c/aria-at-app/issues/1143 here?

Reply:
Agree. Done!